src/Angor/Client/Pages/Wallet.razor

```diff
@@ -1758,16 +1770,27 @@ else
         isPassphraseEnabled = false;
         backupConfirmation = false;
         showBackupConfirmationError = false;
-        
+
         // Reset verification state
         selectedWordsToVerify.Clear();
         userVerificationInputs.Clear();
         verificationPassed = false;
         showVerificationError = false;
     }
 
+    private void ConvertAmountToSatoshis(ChangeEventArgs e)
+    {
+        if (decimal.TryParse(e.Value?.ToString(), out var amount))
+        {
+            _sendInfo.SendAmount = Money.Coins(amount).Satoshi;
+        }
+    }
 
+    private decimal sendAmount
```

Comment:
this is a good trick to bypass the bind value being presented in btc on the page